### ISSUE TITLE
main cards on the 'item' and 'seal description' views

### ISSUE THREAD

**Issue by johnmcewan** [thehabes/DigiSig#76]
There is a 'main' card on both the 'seal description' and 'item' page. It appears at the top under the title and permalink information. This card only appears on its own so this leaves considerable empty space on the right hand side of the screen. Could the horizontal dimensions of these cards be adjusted so that they can expand horizontally to make better use of the available space?

**Comment by thehabes** [comment]
Yes, these cards are special and I can catch and identify their creation as such in the code.  I have given them special classing, we can alter the look however we want.

**Comment by thehabes** [comment]
I have made them fairly wide (80% screen width) and allow the text within them to flow to as wide as the container, instead of forcing the 250px max.
